App.js

```diff
@@ -0,0 +1,37 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const cors = require("cors");
+const app = express();
+const session = require('express-session');
+const db = require("./models");
+const posts = require('./routes/posts.routes');
+const users = require('./routes/users.routes');
+
+require('dotenv').config();
+
+// db.sequelize.sync({ force: true }).then(() => {          
+//     console.log( "Drop and re-sync db.");
+// }); 
+
+app.use(session({
+    secret: "thisismysecrctekey",
+    saveUninitialized: true,
+    cookie: { maxAge: 1000 * 60 * 60 * 24 },
+    resave: false
+    },));
```

Comment:
We are developing an api so sessions don't make any sense. Please create authentication with jwt & passport